src/phoenix/server/main.py

```diff
@@ -88,6 +91,33 @@ def _get_pid_file() -> Path:
     return get_pids_path() / str(os.getpid())
 
 
+def _get_tracer_provider(
+    *,
+    http_endpoint: Optional[str] = None,
+    grpc_endpoint: Optional[str] = None,
```

Comment:
Should we make these have `Optional[List[str]]` types in case someone want to log to more than one collector?

Reply:
mmm...no.